utils/umaApi.ts

```diff
@@ -198,17 +202,18 @@ export async function fetchCompleteSynth<T extends { type: ContractType }>(
     const stats = await client.getSynthStats(synth.address);
     const state = await client.getState<T>(synth.address);
     const lastTvl = await client.getLatestTvl(synth.address);
-    const [{ value: ydayTvl }] = await client.request(
+    const [{ value: ydayTvl = NaN } = {}] = await client.request(
       "tvlHistorySlice",
```

Comment:
i think this needs to change to `global/tvlHistorySlice`.  I'm afraid to make the root path global since we dont have a clean way of doing this without breaking production